nixos/modules/hardware/video/nvidia.nix

```diff
@@ -172,6 +172,11 @@ in
     environment.systemPackages = [ nvidia_x11.bin nvidia_x11.settings ]
       ++ lib.filter (p: p != null) [ nvidia_x11.persistenced ];
 
+    systemd.tmpfiles.rules = [
+      "L+ /run/nvidia-docker/bin - - - - ${nvidia_x11.bin}/origBin"
```

Comment:
Is this useful for all users?

Reply:
No, you are right. I have gated it with the same flag.